### PR TITLE
final part 1

### PR DESCRIPTION
**Pull request by NMandel527**

### CODE REVIEW

**Comment by andrewoid** [comment]
How do you have merge conflicts?

File: build.gradle [hunk 1]
```diff
@@ -1,9 +1,60 @@
 plugins {
-    id 'java'
+    id 'java-library'
+    id "com.vanniktech.maven.publish" version "0.28.0"
```
Comment (andrewoid) [inline build.gradle]: You were never told to add any of this

File: src/main/java/mandel/museum/ArtObjects.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package mandel.museum;
+
+public class ArtObjects
```
Comment (andrewoid) [inline src/main/java/mandel/museum/ArtObjects.java]: Singular, not plural

File: src/main/java/mandel/museum/RijksService.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
+package mandel.museum;
+
+import io.reactivex.rxjava3.core.Single;
+import retrofit2.http.GET;
+import retrofit2.http.Query;
+
+public interface RijksService
+{
+    @GET("/api/en/collection")
+    Single<Art> artP(
```
Comment (andrewoid) [inline src/main/java/mandel/museum/RijksService.java]: Change these methods names so they mean something

File: src/test/java/mandel/museum/RijksServiceFactoryTest.java [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        RijksService service = new RijksServiceFactory().getService();
+
+        //when
+        Art artP = service.artP(
+                apiKey.get(),
+                27
+        ).blockingGet();
+
+        //then
+        assertNotNull(artP.artObjects[0]);
```
Comment (andrewoid) [inline src/test/java/mandel/museum/RijksServiceFactoryTest.java]: Test what is inside the artobject

**Comment by NMandel527** [comment]
> How do you have merge conflicts?
It's not letting me reply directly, but I was just looking at the merge conflict. I'm not sure what happened. It was fine and then I updated it by adding a readme and this came up.

**Comment by andrewoid** [comment]
> > How do you have merge conflicts?
> It's not letting me reply directly, but I was just looking at the merge conflict. I'm not sure what happened. It was fine and then I updated it by adding a readme and this came up.
> 

How did you add a Readme?

File: build.gradle (hunk 1 above)
Comment (NMandel527) [reply build.gradle]: I was looking at the code in the src file that was sent with the information about apis. Should I delete the code that I added from there?

Comment (andrewoid) [reply build.gradle]: You should. It will screw up everything else.

**Comment by NMandel527** [comment]
> > > How do you have merge conflicts?
> > > It's not letting me reply directly, but I was just looking at the merge conflict. I'm not sure what happened. It was fine and then I updated it by adding a readme and this came up.
> 
> How did you add a Readme?

I added a new file called README.md and then I added information to it. Then I did git add . git commit -m "added readme" git push origin api

**Comment by andrewoid** [comment]
> > > > It's not letting me reply directly, but I was just looking at the merge conflict. I'm not sure what happened. It was fine and then I updated it by adding a readme and this came up.

It doesnt tell me what the conflict is

File: build.gradle (hunk 1 above)
Comment (NMandel527) [reply build.gradle]: Ok will do

**Comment by NMandel527** [comment]
> > > > > It's not letting me reply directly, but I was just looking at the merge conflict. I'm not sure what happened. It was fine and then I updated it by adding a readme and this came up.
> 
> It doesnt tell me what the conflict is

This is all it's telling me. Is there a way for me to find out more?
Screenshot 2024-05-15 225142

**Comment by andrewoid** [comment]
> > > > > > It's not letting me reply directly, but I was just looking at the merge conflict. I'm not sure what happened. It was fine and then I updated it by adding a readme and this came up.
> > 
> > It doesnt tell me what the conflict is
> 
> This is all it's telling me. Is there a way for me to find out more?
> Screenshot 2024-05-15 225142
> 

You and I will need to meet tomorrow and resolve this.

**Comment by NMandel527** [comment]
> > > > > > > It's not letting me reply directly, but I was just looking at the merge conflict. I'm not sure what happened. It was fine and then I updated it by adding a readme and this came up.
> > > 
> > > 
> > > It doesnt tell me what the conflict is
> > 
> > 
> > This is all it's telling me. Is there a way for me to find out more?
> > Screenshot 2024-05-15 225142
> 
> You and I will need to meet tomorrow and resolve this.

Ok thank you. What time works best?

**Comment by andrewoid** [comment]
Email me in the morning.

**Comment by NMandel527** [comment]
> Email me in the morning.

Ok thank you so much

**Review by andrewoid — APPROVED** [review]

**Pull request merged**